pkg/sql/logictest/testdata/logic_test/generators

```diff
@@ -8,6 +8,20 @@ generate_series
 2
 3
 
+query T colnames
+SELECT * FROM GENERATE_SERIES('2017-11-11 00:00:00'::TIMESTAMP, '2017-11-11 03:00:00'::TIMESTAMP, '1 hour')
+----
+generate_series
+2017-11-11 00:00:00 +0000 +0000
+2017-11-11 01:00:00 +0000 +0000
+2017-11-11 02:00:00 +0000 +0000
+2017-11-11 03:00:00 +0000 +0000
+
+
+query T
+SELECT * FROM GENERATE_SERIES('2017-11-11 00:00:00'::TIMESTAMP, '2017-11-11 03:00:00'::TIMESTAMP, '-1 hour')
+----
+
```

Comment:
Please add another test to check that the right thing will be returned if the interval is negative and the second argument is less than the first.

Reply:
ok